src/Examples/Deposit/ConsolidateTransactionEVMExample.ts

```diff
@@ -0,0 +1,60 @@
+import { CoinsEnum, LiminalEnvironment, LiminalJs, Wallet } from "@lmnl/liminaljs";
+import { Guid } from "guid-typescript";
+import { ConsolidatedTransactionAsync } from "../../Helpers/ConsolidateTransaction";
+import { LiminalAuthAsync } from "../../Helpers/LiminalAuth";
+import { WalletInstanceAsync } from "../../Helpers/WalletInstance";
+import { clientId, clientSecretId, tsmCred } from "../../Settings";
+
+
+export const main=async():Promise<void>=>{
+
+    try
+    {
+        // Step 1: Auth
+        let liminalJs:LiminalJs=await LiminalAuthAsync({
+            liminalOptions:{
+                clientId:clientId,
+                clientSecret: clientSecretId
+            },
+            env:LiminalEnvironment.test
+        });
+
+        // Step 2: Get Wallet Instance
+        let walletInstance:Wallet=await WalletInstanceAsync({
+            liminalJs:liminalJs,
+            coin:CoinsEnum.eth,
+            walletId:0
+        });
+
+        // Step 3: Consolidate Send Transaction
+
+        let transactionResponse:any[]=await ConsolidatedTransactionAsync({
```

Comment:
Need to take consolidation time in env in hours
Script shold run as per input 
If 5 is set in env , consolidate should run every 5 hours